aodndata/srs/srs_ghrsst.py

```diff
@@ -2,19 +2,21 @@
 import re
 from datetime import datetime
 
+from aodncore.util.misc import get_pattern_subgroups_from_string
+
 from aodncore.pipeline import HandlerBase
 from aodncore.pipeline.exceptions import InvalidFileNameError
 
 L3S_L3C_FILE_PATTERN = re.compile(r"""
-                                ^(?P<nc_time_cov_start>[0-9]{14})-ABOM-
+                                (.*/|)(?P<nc_time_cov_start>[0-9]{14})-ABOM-
```

Comment:
What was the reason for this change to the pattern? Looks a bit odd to me...

Reply:
https://github.com/aodn/python-aodndata/pull/52/files#diff-62a887b1575ecffb84e3cff79877c85eR42

But you are already grabbing the basename here for this purpose?

Reply:
good point, but I should change it here 
https://github.com/aodn/python-aodndata/pull/52/files#diff-62a887b1575ecffb84e3cff79877c85eR45